### PR TITLE
Add Groona::Object#id_accessor?

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: ext/groonga/rb-grn-object.c [hunk 1]
```diff
@@ -1895,6 +1895,40 @@ rb_grn_object_key_accessor_p (VALUE self)
     return CBOOL2RVAL(key_accessor_p);
 }
 
+/*
+ * Checks whether the object is id accessor or not.
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]:
```text
id -> `_id`
```

File: ext/groonga/rb-grn-object.c [hunk 2]
```diff
@@ -1895,6 +1895,40 @@ rb_grn_object_key_accessor_p (VALUE self)
...
+ *   users = Groonga["Users"]
+ *   users.column("_id").id_accessor? # => true
+ *
+ * @example `false` case: `column("_key")` is an accessor.
+ *   Groonga::Schema.create_table("Users", :key_type => :short_text)
+ *   users = Groonga["Users"]
+ *   users.column("_key").key_accessor? # => false
+ *
+ * @overload id_accessor?
+ *   @return [Boolean] `true` if the object is id accessor,
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: ditto.

File: ext/groonga/rb-grn-object.c [hunk 3]
```diff
@@ -1895,6 +1895,40 @@ rb_grn_object_key_accessor_p (VALUE self)
     return CBOOL2RVAL(key_accessor_p);
 }
 
+/*
+ * Checks whether the object is id accessor or not.
+ *
+ * @example `true` case: `column("_id")` is an accessor.
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: Is "is an accessor." needed?

File: ext/groonga/rb-grn-object.c [hunk 4]
```diff
@@ -1895,6 +1895,40 @@ rb_grn_object_key_accessor_p (VALUE self)
...
+ * @overload id_accessor?
+ *   @return [Boolean] `true` if the object is id accessor,
+ *     `false` otherwise.
+ */
+static VALUE
+rb_grn_object_id_accessor_p (VALUE self)
+{
+    grn_ctx *context;
+    grn_obj *object;
+    grn_bool id_accessor_p = GRN_FALSE;
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: We should use `bool` and `false` instead.

File: ext/groonga/rb-grn-object.c (hunk 3 above)
Comment (komainu8) [reply ext/groonga/rb-grn-object.c]: This is needless.
I've removed it.

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

File: ext/groonga/rb-grn-object.c [hunk 5]
```diff
@@ -1895,6 +1895,40 @@ rb_grn_object_key_accessor_p (VALUE self)
...
+ *
+ * @example `true` case: `column("_id")`.
+ *   Groonga::Schema.create_table("Users", :key_type => :short_text)
+ *   users = Groonga["Users"]
+ *   users.column("_id").id_accessor? # => true
+ *
+ * @example `false` case: `column("_key")`.
+ *   Groonga::Schema.create_table("Users", :key_type => :short_text)
+ *   users = Groonga["Users"]
+ *   users.column("_key").key_accessor? # => false
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: `id_accessor?`

**Pull request merged**